packages/types/src/definitions/TimerType.type.ts

```diff
@@ -3,4 +3,5 @@ export enum TimerType {
   CountUp = 'count-up',
   TimeToEnd = 'time-to-end',
   Clock = 'clock',
+  External = 'external',
```

Comment:
I am not sure about reusing this. 
The existing behaviour changes the way the time is presented.

I feel the external source is unrelated and I am not sure I would know to look for it here.

On the other hand, we can't provide a countdown countup to an event we don't have control over. 

Not really sure. What do you think?

Reply:
yes i'm also not sure

TimeToEnd and External can't really work at the same time
Clock and External doesn't make much sense either (I think?)

maybe both External Up and Down could be useful to some one 
although I expect that most people would end up using CountDown with external..?

I think maybe it is OK to say that External will always be a CountDown 
or we could add an ExternalUp and ExternalDown
(^maybe one of thees for v2)
or a separate toggle

and then we can revisit for v3,

Reply:
How about a single toggle "source: external" which when is set we disable the timer type in the interface?